build-scripts/gh-actions-setup-inv

```diff
@@ -6,6 +6,8 @@ EOF
 "
 sudo chmod 755 /usr/sbin/policy-rc.d
 sudo apt-get update
+# Avoid GRUB related errors due to runner image configuration by reinstalling it.
+sudo apt-get purge --allow-remove-essential grub\* && sudo apt-get -qq -y install grub-efi && sudo apt-get autoremove && sudo update-grub
```

Comment:
Would it be possible not to reinstall grub at all, by any chance? :-)

Reply:
Makes sense. Let me have a look.